lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -359,6 +358,24 @@ sub log_entry {
     my $get_text =
         sub { my $color = shift // 0; $text_data{$color} //= $self->format_line($data, {color => $color}) . "\n"; return $text_data{$color}; };
 
+    # remove substitution context from message
+    if ($data->{message})
```

Comment:
need some unit tests.

Reply:
could it also be a one line if ?
```suggestion
    $data->{message} =~ s/\".*// if $data->{message}
```

Reply:
why does it remove everything from the first `"` occurence?